how-to/releaseduty/merge-duty/merge_duty.rst

```diff
@@ -153,7 +153,7 @@ Merge beta to release
    run <#do-migration-no-op-trial-runs>`__ one more time, and show the
    diff to another person on releaseduty.
 3. The diff for ``release`` should be fairly similar to
-   `this <https://hg.mozilla.org/releases/mozilla-release/rev/118aa10ac456d05606f113ade5c26ae4637081ce>`__,
+   `this <https://firefoxci.taskcluster-artifacts.net/Aj0GVg5HQ7eS5SdKm4faFg/0/public/logs/release.diff>`__,
```

Comment:
Can we point at something that won't expire?

Reply:
Oh, yes...that would be smart! I'll update this PR after the real merge.